js/script.js

```diff
@@ -0,0 +1,43 @@
+const header = document.querySelector('header');
+const hamburger = document.querySelector('.menu-bars');
+const icon = document.querySelector('.menu-bars i');
+
+// Classes for close menu
+const classNames = ['fa-bars', 'nav-link'];
+
+const removeActive = () => {
+  // console.log('removeActive');
+  header.classList.remove('active');
+  icon.classList.remove('fa-x');
+  icon.classList.add('fa-bars');
+};
+
+const addActive = () => {
+  // console.log('addActive');
```

Comment:
Kindly, I suggest not leaving a comment message in your final files, because your code does not look clean.